src/less/expand-button/ds4/expand-button.less

```diff
@@ -8,27 +8,46 @@
 /* stylelint-disable  no-descending-specificity, no-duplicate-selectors */
 
 button.expand-btn {
+    background-color: @button-regular-background-color;
+    border-color: @button-regular-border-color;
+    color: @button-regular-color;
     line-height: 1.2em;
     padding: 11px 16px;
 
+    &:active {
+        border-color: @button-regular-active-color;
+    }
+
     &[disabled],
     &[aria-disabled="true"] {
         opacity: 0.5;
     }
 }
 
-button.expand-btn svg {
-    height: 8px;
-    width: 8px;
-}
-
 button.expand-btn--regular {
     &:hover,
     &:focus {
         border-color: @color-core-gray-spanish;
     }
 }
 
+button.expand-btn--regular[disabled],
+button.expand-btn--regular[aria-disabled="true"] {
```

Comment:
Would there ever be a case where you want aria-disabled true but button to not be disabled? Or vice-versa? Because there's no point in having both of these cases correct since when a button is disabled you should also have it as aria-disabled right?

Reply:
Yes, I think the case you are referring to is a "partially" disabled button. See here: https://github.com/eBay/skin/blob/master/docs/_includes/common/button.html#L89